yaml/morphology/extension.yaml

```diff
@@ -16,9 +16,9 @@ values:
     brief: spatiotemporal end
   - abbrev: GRA
     name: graduative
-    brief: increasing degree or quantity
+    brief: gradual/fading spatiotemporal beginning
   - abbrev: DPL
     name: depletive
-    brief: decreasing degree or quantity
+    brief: gradual/fading spatiotemporal end
```

Comment:
> The graphic to the right illustrates the spatio-temporal relationship of a noun or verbal concept in the DEPLETIVE to the  context at-hand (i.e., the spatio-temporal “present”). Examples: [...] ‘the mouth of a river’[...]‘the last vestiges of a situation’


Again, by my reading these examples all refer to the _lessening in degree or magnitude_, rather than to a _fading in focus_ on the entity. Again, I agree it's maybe up for debate, and maybe both should be covered.

Reply:
What do you mean by "fading in focus"? It's more than just lessening in degree, because it also signifies an end to the entity (at least according to my understanding).